core/src/main/java/jenkins/model/JenkinsLocationConfiguration.java

```diff
@@ -50,6 +50,17 @@
     public static @CheckForNull JenkinsLocationConfiguration get() {
         return GlobalConfiguration.all().get(JenkinsLocationConfiguration.class);
     }
+    
+    /**
+     * Gets local configuration. For explanation when it could die, see {@link #get()}
+     */
+    public static @Nonnull JenkinsLocationConfiguration getOrDie(){
+        JenkinsLocationConfiguration config = JenkinsLocationConfiguration.get();
+        if (config == null) {
+            throw new IllegalStateException("JenkinsLocationConfiguration instance is missing. Probably the Jenkins instance is fully loaded at that time.");
```

Comment:
typo?  "Probably the Jenkins instance is fully loaded at that time"

Don't you mean **not** fully loaded  (also should be this time)